addons/website_slides_survey/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import models
```

Comment:
I think you can also remove this module, as awa will merge it.

Reply:
Done

Reply:
\o/